src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -408,9 +408,14 @@ private synchronized int execMatlabCommand(Run<?, ?> build, FilePath workspace,
             TaskListener listener, boolean isLinuxLauncher)
             throws IOException, InterruptedException {
         setEnv(build.getEnvironment(listener));
-        //Copy MATLAB scratch file into the workspace
-        copyMatlabScratchFileInWorkspace(MATLAB_RUNNER_RESOURCE, MATLAB_RUNNER_TARGET_FILE,
-                workspace, getClass().getClassLoader());
+        final String testRunMode = this.getTestRunTypeList().getDescriptor().getDisplayName();
```

Comment:
IS there a way we don't have to look at the display name for this? This seems like it is coupling the view with what should be a model concern.